generator/src/FluentStageFactoryGenerator.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\CodeGenerator;
+
+use MongoDB\BSON\Decimal128;
+use MongoDB\BSON\Document;
+use MongoDB\BSON\Int64;
+use MongoDB\BSON\PackedArray;
+use MongoDB\BSON\Serializable;
+use MongoDB\BSON\Timestamp;
+use MongoDB\BSON\Type;
+use MongoDB\Builder\Expression\ArrayFieldPath;
+use MongoDB\Builder\Expression\FieldPath;
+use MongoDB\Builder\Expression\ResolvesToArray;
+use MongoDB\Builder\Expression\ResolvesToObject;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Stage;
```

Comment:
This refers to [src/Builder/Stage.php](https://github.com/mongodb/mongo-php-builder/blob/0.1/src/Builder/Stage.php). But later on, you also have a reference to `Stage\FactoryTrait`, which seems to piggy-back this `use` statement. Is there any reason not to import that trait directly?

I forgot that [namespaces can be imported](https://www.php.net/manual/en/language.namespaces.importing.php), as it seldom comes up in my work.